src/structures/interface/TextBasedChannel.js

```diff
@@ -353,16 +353,17 @@ class TextBasedChannel {
   }
 
   /**
-   * Bulk delete given messages.
+   * Bulk delete given messages from less than two weeks ago.
    * <warn>This is only available when using a bot account.</warn>
    * @param {Collection<string, Message>|Message[]|number} messages Messages to delete, or number of messages to delete
+   * @param {boolean} [filterOld=false] Filter messages older than two weeks automatically
```

Comment:
`Whether to filter the messages to remove those which are older than two weeks automatically`